### PR TITLE
Add docs for setup as SiK telemetry link

### PR DESCRIPTION
**Pull request by olliw42**
title says it

### CODE REVIEW

File: docs/SETUP_SIK.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# mLRS Documentation: Setup as SiK Telemetry Replacement #
+
+([back to main page](../README.md))
+
+This page describes how to set up an mLRS system to work as a bi-directional serial MAVLink telemetry link, independent on any radio, where mLRS' rc data capability is not desired and mLRS should only transmits and receives serial MAVLink data. This setup finds application in situations where traditionally, e.g., SiK telemetry uints would be used.
```
Comment (jlpoltrack) [inline docs/SETUP_SIK.md:5]: How about 'This page describes how to use mLRS as a bi-directional MAVLink telemetry link similar to a SiK telemetry unit.  This setup doesn't require a radio and will only transmit and receive MAVLink data'

File: docs/SETUP_SIK.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+The configuration for this mode of operation just follows that for any other configuration, except that one simply ignores the rc data input on the mLRS Tx module and the rc data output on the mLRS receiver.
+
+For this setup, the mLRS Tx module should be set to "Tx Ch Soure = none". 
+
+There is no specific configuration of the mLRS receiver neccessary. 
+
+Except of "Tx Ch Soure = none", the configuration can exactly follow the settings described in [CRSF Telemetry and Yaapu Telemetry App](CRSF.md), with the obvious modification, that settings related to the RC data are ignored. For instance, the sub chapter [Setup ArduPilot: CRSF receiver](CRSF.md/#crsf-receiver) is to be substituted by the setup required for the used RC system. 
+
+The settings in these sub chapters can be applied directly: [ArduPilot Setup: MAVLink Serial Port](CRSF.md/#mavlink-serial-port), [ArduPilot Setup: Stream Rates](CRSF.md/#stream-rates) and [mLRS Rx Module Setup](CRSF.md/#mlrs-rx-module-setup).
+
```
Comment (jlpoltrack) [inline docs/SETUP_SIK.md:20]: 4 lines of whitespace?

File: docs/SETUP_SIK.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+([back to main page](../README.md))
+
+This page describes how to set up an mLRS system to work as a bi-directional serial MAVLink telemetry link, independent on any radio, where mLRS' rc data capability is not desired and mLRS should only transmits and receives serial MAVLink data. This setup finds application in situations where traditionally, e.g., SiK telemetry uints would be used.
+
+<img src="images/mLRS-docu-setup-sik-telemetry-01.jpg" width="800px">
+
+## Setup
+
+The configuration for this mode of operation just follows that for any other configuration, except that one simply ignores the rc data input on the mLRS Tx module and the rc data output on the mLRS receiver.
```
Comment (jlpoltrack) [inline docs/SETUP_SIK.md:11]: How about 'The configuration for this mode is similar to others, except that one can ignore the rc settings on both the Tx and Rx.'

File: docs/SETUP_SIK.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+For this setup, the mLRS Tx module should be set to "Tx Ch Soure = none".
```
Comment (jlpoltrack) [inline docs/SETUP_SIK.md:13]: Spelling, 'Source' instead of 'Soure'

File: docs/SETUP_SIK.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+## Setup
+
+The configuration for this mode of operation just follows that for any other configuration, except that one simply ignores the rc data input on the mLRS Tx module and the rc data output on the mLRS receiver.
+
+For this setup, the mLRS Tx module should be set to "Tx Ch Soure = none". 
+
+There is no specific configuration of the mLRS receiver neccessary. 
+
+Except of "Tx Ch Soure = none", the configuration can exactly follow the settings described in [CRSF Telemetry and Yaapu Telemetry App](CRSF.md), with the obvious modification, that settings related to the RC data are ignored. For instance, the sub chapter [Setup ArduPilot: CRSF receiver](CRSF.md/#crsf-receiver) is to be substituted by the setup required for the used RC system.
```
Comment (jlpoltrack) [inline docs/SETUP_SIK.md:17]: How about 'With the exception of "Tx Ch Source = none", the configuration can exactly follow the settings described in [CRSF Telemetry and Yaapu Telemetry App](CRSF.md).  If you are using a separate RC system (e.g. ELRS, FrSky, etc.), the sub chapter [Setup ArduPilot: CRSF receiver](CRSF.md/#crsf-receiver) will need to be modified for the system you are using.'

File: docs/SETUP_SIK.md (hunk 1 above)
Comment (olliw42) [reply docs/SETUP_SIK.md]: done

File: docs/SETUP_SIK.md (hunk 2 above)
Comment (olliw42) [reply docs/SETUP_SIK.md:16]: gone

File: docs/SETUP_SIK.md (hunk 3 above)
Comment (olliw42) [reply docs/SETUP_SIK.md]: done

File: docs/SETUP_SIK.md (hunk 4 above)
Comment (olliw42) [reply docs/SETUP_SIK.md]: done

File: docs/SETUP_SIK.md (hunk 5 above)
Comment (olliw42) [reply docs/SETUP_SIK.md]: done

**Comment by olliw42** [comment]
fantastic suggestions, makes it really better.

**Comment by jlpoltrack** [comment]
Looks good to me - will merge if you're good as well.

**Comment by olliw42** [comment]
LGTM

MANY THX !!

**Pull request merged**